docs/posts/ibis-duckdb-geospatial/index.qmd

```diff
@@ -0,0 +1,241 @@
+---
+title: "Ibis + DuckDB geospatial: a match made on Earth"
+author: Naty Clementi
+date: 2023-12-07
+categories:
+    - blog
+    - duckdb
+    - geospatial
+---
+
+Ibis now has support for [DuckDB geospatial functions](https://gist.github.com/ncclementi/fbc5564af709e2d7f8882821e3a8649f)!
+
+This blogpost showcases some examples of the geospatial API for the DuckDB backend. The material is inspired in
+the ["DuckDB: Spatial Relationships"](https://geog-414.gishub.org/book/duckdb/07_spatial_relationships.html) lesson from
+[Dr. Qiusheng Wu](https://geog-414.gishub.org/book/preface/instructor.html)'s course "Spatial Data Management" from the
+Department of Geography & Sustainability at the University of Tennessee, Knoxville.
+
+::: {.callout-note}
+You can check Dr. Qiusheng Wu's full Spatial Data Management course material on its
+[website](https://geog-414.gishub.org/index.html), and the classes are also on
+[YouTube](https://www.youtube.com/watch?v=A4TOAdsXsEs&list=PLAxJ4-o7ZoPe9SkgnophygyLjTDBzIEbi).
+:::
+
+## Data
+
```

Comment:
Maybe a quick note here explaining what we're downloading.

Reply:
added something, let me know if that is good enough. I took the data from the lessons I was following but there was not a lot of detail about it there.